scripts/python/services/GithubService.py

```diff
@@ -13,3 +15,17 @@ def get_outside_collaborators_login_names(self) -> list[str]:
         outside_collaborators = self.client.get_organization(
             self.organisation_name).get_outside_collaborators() or []
         return [outside_collaborator.get("login") for outside_collaborator in outside_collaborators]
+
+    def close_expired_issues(self, repository_name: str):
```

Comment:
A few things to note with the refactors:

- Removed exception handling - prevent us from failing silently
- Adding the `or []` to mitigate any issues with `None/null` return values from client
- Added all conditionals into one if statement for conciseness
- There is a couple of [Magic Numbers/Strings](https://en.wikipedia.org/wiki/Magic_number_(programming)) embedded in the function. It would be preferable for these to be in some sort of config file to decouple the business logic. Chosen to leave this refactoring for later.

Reply:
Adding `or []` to mitigate any issues with `None/null` is a great idea

Reply:
Agree with a seperate file for magic numbers and strings, did the same with the Ruby code app